source/sass/blocks/example.scss

```
@@ -11,14 +15,16 @@
   padding-bottom: 35px;
 
   @media (min-width: $tablet-width) {
-    margin-bottom: 59px;
+    padding-top: 26px;
+    padding-bottom: 59px;
```

Не понимаю почему мы это упустили, но должен быть отступ больше

```suggestion
    padding-bottom: 97px;
```